src/CONST/index.ts

```diff
@@ -1284,6 +1284,7 @@ const CONST = {
                 SHARE: 'SHARE', // OldDot Action
                 STRIPE_PAID: 'STRIPEPAID', // OldDot Action
                 SUBMITTED: 'SUBMITTED',
+                AUTO_SUBMITTED: 'AUTOSUBMITTED',
```

Comment:
Sorry for confusion, there is no `AUTOSUBMITTED` action it's just a `SUBMITTED` action that we will be looking for.